assets/scss/base/_helpers.scss

```diff
@@ -373,3 +373,18 @@
   max-width: 125px;
 }
 
+/*
+Equally spaced out container element
+
+Experimental: Equal top and bottom spacing for container element
+
+Markup:
+<div class="{{modifier_class}}">Content</div>
+
+.equal-spacing - equal top and bottom spacing for container element
```

Comment:
no need for the modifier here (as there is just one example) - the description can be placed in the main description